pkg/distro/generic/distro.go

```diff
@@ -115,6 +113,10 @@ func (d *distribution) Name() string {
 	return d.DistroYAML.Name
 }
 
+func (d *distribution) Aliases() []string {
+	return d.DistroYAML.Aliases
+}
```

Comment:
`Aliases()` on the distro level should be removed, since it is not used or required at this point as far as I can tell.